test/algorithms/test_min_eigen_optimizer.py

```diff
@@ -353,6 +359,38 @@ def test_samples_vqe(self, simulator):
         self.assertAlmostEqual(results.raw_samples[0].fval, opt_sol)
         self.assertEqual(results.raw_samples[0].status, success)
 
+    @data("vqe", "qaoa")
+    def test_runtime(self, subroutine):
+        """Test vqe and qaoa runtime"""
+        optimizer = {"name": "SPSA", "maxiter": 100}
+        backend = QasmSimulatorPy()
+        provider = FakeRuntimeProvider()
+
+        if subroutine == "vqe":
+            ry_ansatz = TwoLocal(5, "ry", "cz", reps=3, entanglement="full")
+            initial_point = np.random.RandomState(42).random(ry_ansatz.num_parameters)
```

Comment:
I suggest using Generator instead of legacy Random State.
https://numpy.org/doc/stable/reference/random/generator.html
```suggestion
            initial_point = np.random.default_rng(42).random(ry_ansatz.num_parameters)
```